includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -113,7 +114,43 @@ function get_columns() {
 
 		return $this->columns;
 	}
-
+	/**
+	 * Append the count value to column headers where applicable
+	 *
+	 * @since  4.2.0
+	 * @access private
+	 *
+	 * @param array $columns Array of columns for the report table.
+	 * @return array The array of columns to use with the table with columns appended to their title
+	 */
+	public function add_totals_to_report_column_headers( array $columns ) {
+		$column_value_map = array();
+		switch ( $this->type ) {
```

Comment:
Instead of adding a switch and adding logic for each type in this method, maybe it would be better to create a specific method for each type and hook that to the same filter? I'm also ok to merge like this and refactor later. 🙂

Reply:
That's a good suggestion! 👍  I'll definitely take a look into it and may be we can all look into a bit more together as we'll need column totals for more tables